src/pytest/test.py

```diff
@@ -2291,4 +2298,6 @@ def testOptimize(env):
     with env.assertResponseError():
         env.assertOk(env.cmd('FT.OPTIMIZE', 'idx', '666'))    
 
-
+def testInfo(env):
+    env.cmd('ft.create', 'idx', 'SCHEMA', 'test', 'TEXT', 'SORTABLE')
```

Comment:
Why do you need to create an index here? Also lets call the test something like: testInfoError